### PR TITLE
EN-65991: publish library automatically

### PR DESCRIPTION
**Pull request by sarah-salinas**
This commit updates the automation for publishing the library. The Jenkins job for the main branch will check for the change and publish the library.

### CODE REVIEW

**Review by rjmac — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -21,7 +22,6 @@ pipeline {
   options {
     ansiColor('xterm')
     buildDiscarder(logRotator(numToKeepStr: '50'))
-    disableConcurrentBuilds(abortPrevious: true)
```
Comment (sarah-salinas) [inline Jenkinsfile:24]: If we abort builds, we could miss a library publish.

**Review by rjmac — APPROVED** [review]

**Comment by sarah-salinas** [comment]
## Testing

https://jenkins-build.socrata.com/job/data-platform/job/soda-fountain/view/change-requests/job/PR-453/6/console - 

It checked out the correct tag / commit for building and publishing the library
```
[Pipeline] { (Publish Library)
[Pipeline] script
[Pipeline] {
[Pipeline] sh
+ git describe --abbrev=0 --match *[0-9].*[0-9].*[0-9]
[Pipeline] sh
+ git config --get remote.origin.url
[Pipeline] checkout
The recommended git tool is: jgit
using credential pipelines-token
Fetching changes from the remote Git repository
Checking out Revision dbe7e1389baf4a6b219081c48d2f628f5488cf3f (refs/tags/v4.0.18)
Commit message: "Setting version to 4.0.18"
First time build. Skipping changelog.
```

**Comment by sarah-salinas** [comment]
## Testing

https://jenkins-build.socrata.com/job/data-platform/job/soda-fountain/view/change-requests/job/PR-453/9/console - 🟢 

It checked out the proper tag / commit
```
+ git describe --abbrev=0 --match *[0-9].*[0-9].*[0-9]
[Pipeline] sh
+ git config --get remote.origin.url
[Pipeline] checkout
The recommended git tool is: jgit
using credential pipelines-token
Fetching changes from the remote Git repository
Checking out Revision dbe7e1389baf4a6b219081c48d2f628f5488cf3f (refs/tags/v4.0.18)
Commit message: "Setting version to 4.0.18"
```

**Comment by sarah-salinas** [comment]
## Restesting after changes

https://jenkins-build.socrata.com/job/data-platform/job/soda-fountain/view/change-requests/job/PR-453/13/console - 🟢 

It checked out the proper tag / commit
```
Checking out Revision dbe7e1389baf4a6b219081c48d2f628f5488cf3f (refs/tags/v4.0.18)
```

**Review by rjmac — APPROVED** [review]

**Pull request merged**